Managers/PlayerManager.gd

```diff
@@ -15,7 +15,8 @@ var player: Player
 var player_position: Vector2i = Vector2i(-1,-1):
 	set(position):
 		player_position = position
-		is_player_initial_position_set = true
+		if player_position.x != -1 and player_position.y != -1:
```

Comment:
is it possible to do `if player_position != (-1, -1)` ?

Reply:
It works! Will refactor